documentation/src/main/asciidoc/topics/con_performance_security.adoc

```diff
@@ -0,0 +1,77 @@
+[id='performance-security_{context}']
+= Securing {brandname} deployments
+
+Protecting your data and preventing network intrusion is arguably the most important aspect of deployment planning.
+Sensitive customer details leaking to the open internet or data breaches that allow hackers to publicly expose confidential information have devastating impacts on business reputation.
+
+With this in mind you need a robust security strategy to authenticate users and encrypt network communication.
+But what are the costs to the performance of your {brandname} deployment?
+How should you approach these considerations during planning?
+
+.Authentication
+
+The performance cost of validating user credentials depends on the mechanism and protocol, as described in the following table:
+
+[%autowidth,cols="1,2,3",stripes=even,%header]
+|===
+|SASL mechansim |HTTP mechanism |Performance impact
+
+|`PLAIN`
+|`BASIC`
+|{brandname} validates credentials once per user over Hot Rod while potentially for every request over HTTP.
+While `PLAIN` and `BASIC` are the fastest authentication mechanisms, they are also the least secure.
+You should only ever use `PLAIN` or `BASIC` in combination with TLS/SSL encryption.
+
+|`DIGEST` and `SCRAM`
+|`DIGEST`
+|For both Hot Rod and HTTP requests, the `DIGEST` scheme uses MD5 hashing algorithms to hash credentials so they are not transmitted in plain text. If you do not enable TLS/SSL encryption then using `DIGEST` is overall less resource intensive than `PLAIN` or `BASIC` with encryption but not as secure because `DIGEST` is vulnerable to monkey-in-the-middle (MITM) attacks and other intrusions.
```

Comment:
monkey-in-the-middle attack?

Reply:
Gender neutral variation of man-in-the-middle. I don't mind too much either way but monkey is a thing.